### DISCUSSION TITLE
Additional categories + specific org agenda views

### DISCUSSION THREAD

**Discussion by cilanadzriddle** [emacs-dashboard/dashboard#530] [Q&A]
Is it possible for me to add more categories? Im looking to show 3 different org agenda views, one that shows TODOs i scheduled for today (which is already the default), another that shows scheduled TODOs from my homework org file that compiles all of my school assignments, and another file of my recurring tasks and chores which is also compiled on another org file

I do acknowledge the possibility that this can be done with some basic e-lisp code, but I just started using emacs 3 months ago and my knowledge in e-lisp is next to nothing (i just copy paste lines of code i see on the internet to my own config file, which i know is bad practice)

Also, I just wanted to say that i really love this package, it motivates me to use emacs more often :))

**Comment by jcs090218** [comment]
I'm not very familiar with the org agenda. Let me ping another maintainer.

cc @ricardoricho

**Reply by cilanadzriddle** [reply]
i managed to find a way by defining a function
```
(defun switch-to-homework ()
  (interactive)
  (setq dashboard-match-agenda-entry
      "TODO=\"HOMEWORK\"")
  (revert-buffer))
```
with my homework headings having a todo entry titled HOMEWORK

now i just need to find a way to bind it to a single-letter command like how clicking "g" while on the dashboard buffer reverts it

**Reply by ricardoricho** [reply]
Hi @cilanadzriddle 
Great solution you found there, I was going to suggest something similar, you also could set `dashboard-filter-agenda-entry` and define different "configurations" for the agenda. I would use `dashboard-refresh-buffer` (thats an alias for `dashboard-open`) instead of `revert-buffer`.  You can bind your function using  `define-key` :
```emacs-lisp
(define-key dashboard-mode-map (kbd "H") #'switch-to-homework)
```

**Reply by cilanadzriddle** [reply]
thanks for the key binding tip :] i tried to figure out how `dashboard-filter-agenda-entry` works by looking at other people's configs, and found nothing helpful :/ 

 i instead resorted to the 'match' one. found a config on a blog that uses it so i copied and modified it to my config..
